packages/core-mobile/app/store/walletConnectV2/listeners/index.ts

```diff
@@ -49,4 +50,9 @@ export const addWCListeners = (startListening: AppStartListening): void => {
     actionCreator: setActiveAccountIndex,
     effect: handleAccountChange
   })
+
+  startListening({
+    matcher: isAnyOf(setAccounts, setActiveAccountIndex),
```

Comment:
feel like setAccounts shouldn't be included since that is only called once at initial login